src/components/atoms/Centered/Centered.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import classNames from "classnames";
+
+import { ContainerClassName } from "types/utility";
+
+import "./Centered.scss";
+
+export const Centered: React.FC<
```

Comment:
The name sounds like a state/boolean, can we think of something else? Maybe `CenterContent`, `CenterChildren` or something else.

Reply:
Haven't managed to come up with other variants. Chose `CenterContent`:) Commit https://github.com/sparkletown/sparkle/pull/1995/commits/5d9989860f08448b0260105f0c054f519b00bd03